common/src/main/scala/explore/common/UserPreferencesQueries.scala

```diff
@@ -331,56 +329,41 @@ object UserPreferencesQueries {
 
   object ElevationPlotPreference:
     def updatePlotPreferences[F[_]: ApplicativeThrow](
-      uid:      User.Id,
-      targetId: Target.Id,
-      site:     Site,
-      range:    PlotRange,
-      time:     TimeDisplay
+      userId: User.Id,
+      site:   Site,
```

Comment:
Maybe we should keep the site bound to the target?

Reply:
It was requested to make it global